includes/admin/class-algolia-admin.php

```diff
@@ -99,6 +100,23 @@ public function enqueue_scripts() {
 		);
 	}
 
+	/**
+	 * Add localize strings to scripts.
+	 *
+	 * @author WebDevStudios <contact@webdevstudios.com>
+	 * @since  2.2.0-dev
+	 */
+	public function localize_scripts() {
+
+		wp_localize_script(
+			'algolia-admin-push-settings-button',
+			'algoliaPushSettingsButton',
+			array(
+				'pushBtnAlert' => __( 'Warning: Pushing settings will override the settings in the Algolia dashboard. Do you want to continue?', 'wp-search-with-algolia' ),
```

Comment:
one last small detail, can we get this updated to use one of the escaping internationalization functions? I think `esc_html__` would be fine.

Reply:
Oh geez! I should have added that. Thanks for pointing this out!